### DISCUSSION TITLE
writing-helpful-error-messages/

### DISCUSSION THREAD

**Discussion by giscus** [Hyeon9mak/Hyeon9mak.github.io#165] [Blog comments]
# writing-helpful-error-messages/

   Google Developers - Writing Helpful Error Messages 문서에서 발췌한 내용들 입니다.   

https://hyeon9mak.github.io/writing-helpful-error-messages/

**Comment by dojinyou** [comment]
좋은 글 감사합니다! 구글 자바 스타일 가이드 링크가 2번 복사되어 URL에 해당하는 페이지가 없는 것으로 나오고 있어 수정되면 좋을 것 같습니다 :)

**Reply by Hyeon9mak** [reply]
도진님! 감사합니다 🙇‍♂️

**Comment by moonkorea00** [comment]
커리어리에서 왔습니다 👍  에러처리할 때 좋은 가이드라인이 될 거 같습니다.

**Comment by jinnybrain** [comment]
잘 읽었습니다. 감사합니다 😊

**Comment by sojukang** [comment]
참고할 부분이 많네요 감사합니다 👍

**Comment by gitchannn** [comment]
완전 유익해요! 감사합니다!_!_!_!_!

**Comment by kmss6905** [comment]
좋은 글 잘 읽었습니다!!
